packages/client/src/pages/pinia.vue

```diff
@@ -73,14 +88,17 @@ onDevToolsClientConnected(() => {
         </div>
       </Pane>
       <Pane flex flex-col>
-        <div :key="selected" h-0 grow overflow-auto p-2 class="no-scrollbar">
+        <div v-if="hasRegisteredModule" :key="selected" h-0 grow overflow-auto p-2 class="no-scrollbar">
           <InspectorState
             v-for="(item, key) in state" :id="key"
             :key="key"
-            inspector-id="pinia"
+            :inspector-id="inspectorId"
             :node-id="selected" :data="item" :name="`${key}`"
           />
         </div>
+        <p v-else p-3>
+          No module registered
+        </p>
```

Comment:
Instead of showing empty state/getters fields, provide the error message.
<img width="371" alt="Screenshot 2024-02-21 at 3 27 00" src="https://github.com/vuejs/devtools-next/assets/22590005/0568d879-d93e-4273-9e10-cc99489c9ab9">